VDF.Core/FFTools/FfmpegEngine.cs

```diff
@@ -68,7 +68,8 @@ static class FfmpegEngine {
 				var data = GetThumbnail(new FfmpegSettings {
 					File = videoFile.Path,
 					Position = TimeSpan.FromSeconds(position),
-					GrayScale = 1
+					GrayScale = 1,
+					Width = width,
```

Comment:
I think that Width should be hardcoded again and not a setting/parameter. I'm not commenting other places where you changed it.

Reply:
Too bad,... but OK.
_Then there is probably no user configurable thumbnail size or a better tooltip zoom planed in near future.
Whereby I don't really need both,... the parameterizable approach would have just allowed it. Or should one make the ffmpeg parameters freely configurable once, one could have docked that also there to adjust the image size with.
At least that were just so my thoughts behind it,..._